File: LeetCode/Easy/2248. Intersection of Multiple Arrays/2026-04-27_Accepted.cpp
class Solution {
public:
  vector<int> intersection(vector<vector<int>>& nums) {
    vector<int> count(1001, 0);
    for (auto arr : nums) {
      for (auto num : arr) {
        count[num]++;
      }
    }
    
    vector<int> ans;
    for (int i = 1; i <= 1000; i++) {
      if (count[i] == nums.size()) {
        ans.push_back(i);
      }
    }
    return ans;
  }
};

File: LeetCode/Easy/2264. Largest 3-Same-Digit Number in String/2026-04-27_Accepted.py
class Solution:
  def largestGoodInteger(self, num: str) -> str:
    for i in range(9, -1, -1):
      three_same_digit_number = str(i) * 3
      if three_same_digit_number in num:
        return three_same_digit_number
    return ''
